### ISSUE TITLE
Commandline options no longer recognized

### ISSUE THREAD

**Issue by brookslz** [jashkenas/docco#412]
In the last month the dependency for Commander is no longer being installed in docco 0.8.0 or 0.7.0

This means that things like setting the layout or output folder from the commandline no longer register with Docco. Docco will only run on the files provided with default options from the commandline.

The install log when it was last working:
```
[..................] / rollbackFailedOptional: verb npm-session f76e00499c3be70
[..................] / rollbackFailedOptional: verb npm-session f76e00499c3be70
[..................] - rollbackFailedOptional: verb npm-session f76e00499c3be70
[..................] - rollbackFailedOptional: verb npm-session f76e00499c3be70
[..................] / fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[..................] / fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[..................] / fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[ .................] | fetchMetadata: sill resolveWithNewModule marked@1.2.7 ch
[ .................] | loadDep:graceful-fs: sill resolveWithNewModule jsonfile@
[       ...........] / extract:docco: verb lock using /root/.npm/_locks/staging
[          ........] \ extract:commander: sill extract commander@>= 0.5.2 extra
[           .......] / extract:fs-extra: sill extract fs-extra@>= 0.6.0 extract
[           .......] - extract:docco: sill extract docco@latest extracted to /u
[           .......] - extract:docco: sill extract docco@latest extracted to /u
[            ......] / refresh-package-json:docco: timing action:finalize Compl

/usr/local/bin/docco -> /usr/local/lib/node_modules/docco/bin/docco
+ docco@0.8.0
added 11 packages from 298 contributors in 0.757s
```

The current non-Commander install log:
```
[..................] / rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] / rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] / rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] - rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] - rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] - rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] - rollbackFailedOptional: verb npm-session 56bbb785640acb3
[..................] | fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[..................] | fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[..................] - fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[..................] / fetchMetadata: sill resolveWithNewModule docco@0.8.0 che
[ .................] | fetchMetadata: sill resolveWithNewModule fs-extra@9.1.0
[ .................] | fetchMetadata: sill resolveWithNewModule fs-extra@9.1.0
[ .................] / fetchMetadata: sill resolveWithNewModule fs-extra@9.1.0
[       ...........] \ extract:docco: verb lock using /root/.npm/_locks/staging
[       ...........] \ extract:docco: verb lock using /root/.npm/_locks/staging
[          ........] / extract:marked: sill extract marked@>= 0.2.7 extracted t
[          ........] - extract:fs-extra: sill extract fs-extra@>= 0.6.0 extract
[           .......] \ extract:docco: sill extract docco@latest extracted to /u
[           .......] \ extract:docco: sill extract docco@latest extracted to /u
[           .......] \ extract:docco: sill extract docco@latest extracted to /u
[            ......] \ finalize:docco: sill doSerial finalize 80
[            ......] | refresh-package-json:docco: timing action:finalize Compl
[            ......] - build:underscore: sill linkStuff underscore@1.12.0 is in

/usr/local/bin/docco -> /usr/local/lib/node_modules/docco/bin/docco
+ docco@0.8.0
added 10 packages from 301 contributors in 1.278s
```

(I cannot get an un-truncated version of these logs since I'm running this inside of CI.  Hopefully this is enough to illustrate the issue.)

**Comment by rjp** [comment]
Got the same "options ignored" issue even though `commander` is correctly installed. That was a one-line fix though.  `commander` is definitely still in `package.json` for 0.8.0 - just forked to fix something else and it's there `>= 0.5.2`.

**Comment by jashkenas** [comment]
Thanks for the report — it looks like commander changed their API. I've pushed a fix out as Docco 0.8.1. Please give that a try and let me know if it works for you.
